doc/setup.md

```diff
@@ -271,21 +256,25 @@ Skyportal uses two distinct ports in its configuration:
 1. Internal Port - Where the application is actually hosted
 2. Public-Facing Port - Where external users access the application
 
-If you need to use different ports (e.g., port 5000 is already in use) add the following changes to `skyportal/config.yaml.defaults` override the defaults:
+If you need to use different ports (e.g., port 5000 is already in use):
+1. Copy `skyportal/config.yaml.defaults` to `skyportal/config.yaml` if you haven't already
```

Comment:
I'd remove the `skyportal/` from the path here. Just to avoid confusing because we have a skyportal directory in the skyportal repo, so folks might look inside of this one instead of just looking in the top directory. That's a very minor concern, so you can also leave that there

Reply:
True, I'll remove it.